server/website/website/db/base/target_objective.py

```diff
@@ -169,15 +158,24 @@ def get_metric_metadata(self, dbms_id, target_objective):
         for target_name, target_instance in self._registry[dbms_id].items():
             if target_name == target_objective:
                 targets_list.insert(0, (target_name, target_instance))
-            else:
-                if target_name != USER_DEFINED_TARGET:
-                    targets_list.append((target_name, target_instance))
         if dbms_id in self._udm_metadatas:
             metadata = targets_list + list(self._udm_metadatas[dbms_id]) +\
                        list(self._metric_metadatas[dbms_id])
```

Comment:
Is this part still useful? As well as udm_registered() and register_udm()?

Reply:
It is useful. Although user defined target objective needs to be selected when creating a session, use defined metrics can still be uploaded from the driver side.  When you first time upload UDMs, you need to register_udm.